snuba/admin/clickhouse/nodes.py

```diff
@@ -29,7 +30,7 @@ def _get_local_nodes(storage_key: StorageKey) -> Sequence[Node]:
             {"host": node.host_name, "port": node.port}
             for node in storage.get_cluster().get_local_nodes()
         ]
-    except (AssertionError, KeyError):
+    except (AssertionError, KeyError, ClickhouseError):
```

Comment:
Aren't you raising a `UndefinedClickhouseCluster` exception?

Reply:
Good catch